src/server.h

```diff
@@ -1540,6 +1544,10 @@ struct redisServer {
     struct malloc_stats cron_malloc_stats; /* sampled in serverCron(). */
     redisAtomic long long stat_net_input_bytes; /* Bytes read from network. */
     redisAtomic long long stat_net_output_bytes; /* Bytes written to network. */
+    redisAtomic long long stat_net_input_bytes_sync_fullresync; /* Bytes read from network during replication for fullresync. */
+    redisAtomic long long stat_net_output_bytes_sync_fullresync; /* Bytes written to network during replication for fullresync. */
+    redisAtomic long long stat_net_input_bytes_sync_cmds; /* Bytes read from network during replication for live command stream. */
+    redisAtomic long long stat_net_output_bytes_sync_cmds; /* Bytes written to network during replication for live command stream. */
```

Comment:
i'm not sure we need separate metrics for full-sync traffic vs command-stream replicatoin.
maybe just two:
`net_repl_input_bytes` and `net_repl_output_bytes`?